src/issues/GAS/unsignedComparison.ts

```diff
@@ -4,7 +4,7 @@ const issue: RegexIssue = {
   regexOrAST: 'Regex',
   type: IssueTypes.GAS,
   title: 'Use != 0 instead of > 0 for unsigned integer comparison',
-  regex: /([a-z,A-Z,0-9]*>.?0|0.?<.?[a-z,A-Z,0-9]*)/g,
+  regex: /(?!pragma\s*solidity)[a-z,A-Z,0-9]*>.?0|0.?<.?[a-z,A-Z,0-9]*/g,
```

Comment:
<img width="1258" alt="Screenshot 2023-01-18 at 3 16 27 PM" src="https://user-images.githubusercontent.com/41673773/213194523-92cabf2f-0747-418f-9883-7f28712fe741.png">

Reply:
I don't think it currently works